src/deluge/gui/l10n/seven_segment.cpp

```diff
@@ -380,6 +373,16 @@ PLACE_SDRAM_DATA Language seven_segment{
         {STRING_FOR_NOTES_CLEARED, "CLEAR"},
         {STRING_FOR_AUTOMATION_CLEARED, "CLEAR"},
 
+        /* Strings Specifically for Automation Instrument Clip View
+        * automation_instrument_clip_view.cpp
+        */
+        {STRING_FOR_AUTOMATION, "AUTO"},
+        {STRING_FOR_COMING_SOON, "SOON"},
```

Comment:
Is it intentional that some strings like STRING_FOR_AUTOMATION_OVERVIEW are not translated in 7SEG?

Reply:
oops, this is targetting the wrong branch